content/docs/topics/chart_template_guide/builtin_objects.md

```diff
@@ -25,6 +25,10 @@ access in your templates.
     upgrade or rollback.
   - `Release.IsInstall`: This is set to `true` if the current operation is an
     install.
+  - `Release.Revision`: The revision number for this release. On install, this is
+    0, and it is incremented with each upgrade and rollback.
```

Comment:
Should the number be 1 on install, as revisions are 1-indexed, or was this what you observed in practice?

Reply:
Oh! You are right!

Reply:
```suggestion
    1, and it is incremented with each upgrade and rollback.
```

Reply:
SGTM